docs/UserGuide.md

```diff
@@ -534,48 +532,66 @@ Examples:
 ![LectureContext](images/LectureContext.png)
 When in a module or lecture context, the `/mod` argument will be injected if only the `/mod` argument is omitted in the original command (refer to [Navigation Injection](#navigation-injection) for more information).
 
-### Delete Module
+### Delete
 
-> Deletes the specified module(s) and all its embodied content from the application
+#### Delete Module
 
-Format: `delete {module_code_1}[, {module_code_2}[, {module_code_3}[, ...]]]`
+> `delete {module_code_1}[, {module_code_2}[, {module_code_3}[, ...]]]`
 
-- `module_code_1`, `module_code_2`, `module_code_3`, ...: Multiple modules can be specified to be deleted by specifying multiple module codes, separating them by commas(",")
-- Module codes must be of valid format
-- If any module specified does not exist, nothing changes within the model
+Deletes the specified module(s) and all its embodied content from the application
+- `module_code_1`, `module_code_2`, `module_code_3`, ... : The codes of modules to be deleted
```

Comment:
I think you should update this to follow the current convention of User Guide of using < span style >:

< span style="color:#e46c0a" >`{module_code_1}[, {module_code_2}[, {module_code_3}[, ...]]]`< /span > : ... (extra space are added)